src/OpenTelemetry.Api/README.md

```diff
@@ -172,16 +172,11 @@ is the .NET `Activity` API. Guidance for instrumenting using this API is
 documented fully in the TBD(dotnet activity user guide link), but is described
 here as well.
 
-1. Install the `System.Diagnostics.DiagnosticSource` package version `5.0.1` or
-   above to your application or library.
+1. Install the latest stable `System.Diagnostics.DiagnosticSource` to your
+   application or library.
 
-    ```xml
-    <ItemGroup>
-      <PackageReference
-        Include="System.Diagnostics.DiagnosticSource"
-        Version="5.0.1"
-      />
-    </ItemGroup>
+    ```shell
+    dotnet add package System.Diagnostics.DiagnosticSource
```

Comment:
If the project already has an older version explicit/implicit dependency of DiagnosticSource, would this command upgrade the package to the latest version or not?

Reply:
it'll upgrade to the latest stable. 
It'll also add this as a explicit dependency, even if the project already had an implicit dependency.

Reply:
Thanks for confirming this!